stacks-common/src/util/log.rs

```diff
@@ -26,6 +26,8 @@ use std::time::{Duration, SystemTime};
 
 lazy_static! {
     pub static ref LOGGER: Logger = make_logger();
+    pub static ref STACKS_LOG_FORMAT_TIME: Result<String, env::VarError> =
```

Comment:
can you use Option here instead of result? as also discussed below.

clarification: is this a lazy assignment?

Reply:
Yes, it's a lazy assignment.